cmd/ui/src/views/Explore/EntityInfo/EntityInfoPanel.tsx

```diff
@@ -32,14 +39,26 @@ interface EntityInfoPanelProps {
 const EntityInfoPanel: React.FC<EntityInfoPanelProps> = ({ selectedNode, sx }) => {
     const styles = usePaneStyles();
     const [expanded, setExpanded] = useState(true);
-    const { setExpandedSections } = useEntityInfoPanelContext();
+    const { setExpandedSections, expandedSections } = useEntityInfoPanelContext();
+    const { expandedPanelSections } = useExploreParams();
+    const { data: backButtonFlag } = useFeatureFlag('back_button_support');
     const previousSelectedNode = usePreviousValue(selectedNode);
 
     useEffect(() => {
-        if (previousSelectedNode?.id !== selectedNode?.id) {
-            setExpandedSections({ 'Object Information': true });
+        if (backButtonFlag?.enabled && previousSelectedNode?.id !== selectedNode?.id && expandedPanelSections) {
+            const initialExpandedSections = { ...formatPanelSectionsParams(expandedPanelSections) };
+            setExpandedSections(initialExpandedSections);
+        } else if (!backButtonFlag?.enabled && previousSelectedNode?.id !== selectedNode?.id) {
+            setExpandedSections({});
```

Comment:
For V1 flows, looks like currently opened accordions stay open when selecting new entities of the same type so I think we still need that first `setExpandedSections({ 'Object Information': true });` still? Something like:

```tsx
  if (previousSelectedNode?.id !== selectedNode?.id) {
            if (backButtonFlag?.enabled && expandedPanelSections) {
                const initialExpandedSections = { ...formatPanelSectionsParams(expandedPanelSections) };
                setExpandedSections(initialExpandedSections);
            } else {
                setExpandedSections({ 'Object Information': true });
            }
        }
```

But take that with a grain of salt, im not sure how the section open by default, `Object Information`, works now

Reply:
disregard - something was off locally